CONTRIBUTING.md

```diff
@@ -25,8 +25,12 @@ downloaded and installed which can be somewhat time consuming. Be patient.
 Note that if you edit a `.proto` file you will need to manually regenerate the associated
 `.pb.{go,cc,h}` files using `go generate`.
 
-To add or update a dependency, use `go get -u` and then
-`glock save github.com/cockroachdb/cockroach` and commit the changes to the GLOCKFILE.
+To add or update a dependency:
+- `go get -u` to updates the dependancies or `go get {package}` to add a new dependancy
```

Comment:
Wow.  I really messed with the english language there.  Clearly didn't proofread.  Fixing.